nautobot/extras/models/models.py

```diff
@@ -680,6 +680,21 @@ class GraphQLQuery(
     SavedViewMixin,
     BaseModel,
 ):
+    # A Graphql Query *may* be owned by another model, such as a GitRepository, or it may be un-owned
+    owner_content_type = models.ForeignKey(
+        to=ContentType,
+        on_delete=models.CASCADE,
+        limit_choices_to=FeatureQuery("grapql_query_owners"),
```

Comment:
Need to add this flag to `nautobot.extras.constants.EXTRAS_FEATURES` and add it as a feature on `nautobot.extras.models.datasources.GitRepository`